README.md

```diff
@@ -22,7 +23,7 @@ Run the bridging experiment as follows:
 
 ```
 $ pipenv run python pmedian.py
-$ julia --project=. pmedian.jl
+$ julia --project=. -Jmoibenchmark.so pmedian.jl
```

Comment:
that is interesting indeed.
I semi copy/pasted this part from the manual.
I always tested this script on windows. So it might not even matter what the extension is there.
Anything particular you would like to test? no extension? `.julia`? `.jll`?

Reply:
I have a `moibenchmark.so` file in the directory. It also works if I use `-Jmoibenchmark`. Does that work on windows?

Reply:
On windows, if I use `-Jmoibenchmark.so` it works, but if I use `-Jmoibenchmark` it doesn't.
If I rename to only `moibenchmark` (no extension) it does not work.
If I rename to `moibenchmark.dll` ti works by using both options: `-Jmoibenchmark` and `-Jmoibenchmark.dll`
If I build with `sysimage_path="moibenchmark",` (no extension) it generates an `.exe` and it does not work.
If I build with `sysimage_path="moibenchmark.jll",` it only works when called as `-Jmoibenchmark.jll`.

I changed to select the system extension automatically:
`sysimage_path="moibenchmark."*Libdl.dlext,`
Now it should work everywhere just by calling `-Jmoibenchmark`